lib/samples/set_reference_scale/README.md

```diff
@@ -0,0 +1,36 @@
+# Set reference scale
+
+Set the map's reference scale and which feature layers should honor the reference scale.
+
+![Image of set reference scale](set_reference_scale.png)
+![Image of set reference scale settings](set_reference_scale_settings.png)
+
+## Use case
+
+Setting a reference scale on an `ArcGISMap` fixes the size of symbols and text to the desired height and width at that scale. As you zoom in and out, symbols and text will increase or decrease in size accordingly. When no reference scale is set, symbol and text sizes remain the same size relative to the `ArcGISMapView`.
+
+Map annotations are typically only relevant at certain scales. For instance, annotations to a map showing a construction site are only relevant at that construction site's scale. So, when the map is zoomed out that information shouldn't scale with the `ArcGISMapView`, but should instead remain scaled with the `ArcGISMap`.
+
+## How to use the sample
+
+* Use the drop-down menu through the Settings dialog to set the map's reference scale (1:500,000 1:250,000 1:100,000 1:50,000).
+* Tap the button to set the map scale to the reference scale.
+* Use the checkboxes in the Settings dialog to set which feature layers should honor the reference scale.
```

Comment:
```suggestion
Tap the "Settings" button to load the settigs dialog. Use the drop-down menu to set the map's reference scale (1:500,000 1:250,000 1:100,000 1:50,000). You can choose which feature layers should honor the reference scale using the checkboxes. Tap the "Set to Reference Scale" button to set the map scale to the reference scale.
```

Appreciate you've copied the design with the steps here but I think the ordering needs changing to make things clearer, as the checkboxes are related to the reference scale, and the button setting the map scale is a different thing. I actually also think the design of the settings panel could also be slightly tweaked to help make this distinction between map and reference scale clearer. Right now it  also feels like I have to use the button to apply the settings.

Maybe have in this order:
- Settings
- A subheading "Reference Scale"
- Dropdown (as-is)
- Text "Apply Reference Scale to Layers"
- Checkboxes (as is)
- A subheading "Map Scale"
- Display the current map scale (see other SDK implementations)
- Button, change text to "Set to Reference Scale"

Reply:
Yes this is what I initially saw for iOS but thought since the common design spec doesn't mention these, maybe they are unnecessary. I will make sure to update the order but should we suggest this change to the common design readme?

Reply:
It's probably not necessary in this case, because the UI is so different for desktop vs mobile. In this case we're needing to make everything quite linear because it's in a popup type dialog, whereas desktop they'd have all the controls on the screen.